lib/datadog/tracing/contrib/grpc/formatting.rb

```diff
@@ -114,7 +114,7 @@ def format_resource_name(grpc_full_method)
             def extract_grpc_service(grpc_full_method)
               parts = grpc_full_method.split('/')
               if parts.length < 3
-                VALUE_UNKNOWN
+                ''
```

Comment:
What is the implication of this change?

Reply:
This is because we don't want to set the standard of setting values for tags that should be empty. Right now, we use the idea that if a tag is empty then the value doesn't exist, but if we set the 'unknown' value then, we have 2 conflicts:

- The user could for some odd reason set the actual value of the tag to 'unknown' which can create a collision
- The existing logic assumes that any tag with a value is a valid defined tag, so we have propagations of the value 'unknown' to other places where in reality, we would want to skip the value.